src/Entities/Transaction.php

```diff
@@ -725,4 +725,100 @@ public function getReferenceData()
     {
         return $this->getAttribute('referenceData');
     }
+
+    /**
+     * @return bool
+     */
+    public function getIsRebill()
+    {
+        return $this->getAttribute('isRebill');
+    }
+
+    /**
+     * @return string
+     */
+    public function get3ds()
+    {
+        return $this->getAttribute('3ds');
```

Comment:
That doesn't correct, it should be an object

Reply:
Added ThreeDSecureResult

Reply:
Attribute factort is missing, resource ThreeDSecureResult won't be created, e.g.:
https://github.com/Rebilly/rebilly-php/blob/c6b8890c6fadb2690f2b45c9af7077d68ecc5869/src/Entities/Transaction.php#L461-L469